packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -466,8 +466,8 @@ contract OPStackManager is ISemver, Initializable {
             disputeGameFactory: address(_output.disputeGameFactoryProxy),
             optimismPortal: address(_output.optimismPortalProxy),
             optimismMintableERC20Factory: address(_output.optimismMintableERC20FactoryProxy),
-            gasPayingToken: Constants.ETHER
-        });
+            gasPayingToken: Constants.ETHER // TODO the production standard config OPSM should not have custom gas token
```

Comment:
Why is this TODO? It seems like it's already satisfied.

Reply:
I thought that the `SystemConfig.Addresses` struct `op-contracts/v1.6.0` did not have a `gasPayingToken` field. Turns out it does, so I've deleted this comment in 86d43ea80

https://github.com/ethereum-optimism/optimism/blob/33f06d2d5e4034125df02264a5ffe84571bd0359/packages/contracts-bedrock/src/L1/SystemConfig.sol#L41